Consul/Operator.cs

```diff
@@ -249,6 +279,23 @@ public Task<QueryResult<string[]>> SegmentList(CancellationToken ct = default)
         {
             return SegmentList(QueryOptions.Default, ct);
         }
+
+        /// <summary>
+        /// CreateArea will create a new network area.
```

Comment:
I think we should leave the part with returned ID though:
```suggestion
        /// CreateArea will create a new network area, a generated ID will be returned on success.
```

Reply:
I deleted that without thinking 😂—thanks for pointing it out!